react/features/base/app/components/BaseApp.js

```diff
@@ -110,7 +111,7 @@ export default class BaseApp extends Component<*, State> {
      * @returns {Promise}
      */
     _initStorage(): Promise<*> {
-        const { _initializing } = window.localStorage;
+        const { _initializing } = jitsiLocalStorage;
```

Comment:
I added `_initializing` method on `DummyLocalStorage` class but I'm not sure if this works if you have another idea let me know,
I've never seen _initializing in `window.localStorage`, how it works?

Reply:
You don't need the `_initializing` method. This should be the equivalent of `localstorage.getItem('_initializing')`.

Reply:
Ooo! I need to change it because this is wrong